app/src/main/res/layout/activity_main.xml

```diff
@@ -21,7 +21,7 @@
         android:id="@+id/logoutBt"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginTop="140dp"
+        android:layout_marginTop="312dp"
```

Comment:
What's the reason for changing the value?

Reply:
While checking the build for the demo, The buttons were all at the same spot, so I had to change some positions.